README.md

```diff
@@ -122,6 +122,12 @@ $ docker exec -it pattern-library-gulp-watch /bin/bash
 elife@...$ node_modules/.bin/gulp test:unit
 ```
 
+To run a particular gulp test:
```

Comment:
Can we make this or create a heading so the test scripts show up better as it's in the middle of the docker setup

Reply:
@discodavey Thanks, it is not suitable section, I moved it on the bottom under section "Running tests".